category-service/src/main/java/com/edmart/category/entity/BaseEntity.java

```diff
@@ -0,0 +1,27 @@
+package com.edmart.category.entity;
+
+import lombok.Data;
+import javax.persistence.MappedSuperclass;
+import javax.persistence.PrePersist;
+import javax.persistence.PreUpdate;
+import java.time.LocalDateTime;
+
+@Data
+@MappedSuperclass
+public class BaseEntity {
+
+    protected LocalDateTime createdAt;
+    protected LocalDateTime updatedAt;
+
```

Comment:
refactor and remove the rest of the code. just leave only the two time fields and their annotations